src/context/like-context.js

```diff
@@ -0,0 +1,46 @@
+import { useReducer, useContext, createContext } from "react";
+
+const likeContext = createContext(null);
+
+const LikeContextProvider = ({ children }) => {
```

Comment:
for maintaining a good naming convention we generally  don't use the `context` word in provide (example: LikeProvider)

Reply:
Oh okay.